src/app/components/wrapper-components/tutorial-game-wrapper/tutorial-game-wrapper.wrapper.component.spec.ts

```diff
@@ -819,17 +822,17 @@ describe('TutorialGameWrapperComponent (wrapper)', () => {
                 componentTestUtils.findElement('#currentMessage').nativeElement.innerHTML;
             expect(currentMessage).toBe(expectedMessage);
         }));
-        it('When unwanted click, and no move done, restart should not be needed', fakeAsync(async() => {
+        it('When unwanted click, and no move done, show restart button with possibility of seeing solution', fakeAsync(async() => {
```

Comment:
it('should show restart button (with possibility of seeing solution) when unwanted click done but no move done') ?